### PR TITLE
adding custom details to event

### LINKED ISSUE #128: Select onChange handler
Hello again, 

Would you mind adding selectedOptions and selectedIndex when calling the event handler ?

```js
this.props.onChange(
  this.MDComponent.selectedOptions,
  this.MDComponent.selectedIndex
);
````

It could let us write more generic code, without ```refs```tricks ;)

```js
selectedChanged = (name) => (values, i) => {
   console.log(name, i, values)
   this.setState({ [name]: i })
}

render() {
 return (
    <Select onChange={this.selectedChanged('chosenOption')}>
    ...
  )
}
```

### PR DESCRIPTION
**Pull request by prateekbh**
this adds the selected index and selected options to the event.'
Fixes #128

### CODE REVIEW

**Comment by prateekbh** [comment]
@cedric-marcone PTAL, if this satisfies what u were looking for

File: sample/home.jsx [hunk 1]
```diff
@@ -151,9 +151,9 @@ export default class Home extends Component {
             this.presel = presel;
```
Comment (cedric-marcone) [inline sample/home.jsx]: Shouldn't we remove the "ref trick" in the example now that it's not needed anymore to extract values ?

**Comment by cedric-marcone** [comment]
LGTM for the update : perfect and future proof, thanks.
But, I added a comment regarding the documentation example for the Select component.

File: sample/home.jsx (hunk 1 above)
Comment (prateekbh) [inline sample/home.jsx]: i'll update the gh-pages

**Pull request merged**